sqlglot/errors.py

```diff
@@ -47,3 +78,7 @@ def concat_errors(errors: t.Sequence[t.Any], maximum: int) -> str:
     if remaining > 0:
         msg.append(f"... and {remaining} more")
     return "\n\n".join(msg)
+
+
+def concat_struct_errors(errors: t.Sequence[ParseError]) -> t.List[t.Dict[str, t.Any]]:
```

Comment:
`struct` is an overloaded word.

How about we call this `merge_errors` and the above `concat_messages`?

Reply:
Thanks, agree those are better names. Fixed in next commit.